src/shims/manual/mscorlib.forwards.cs

```diff
@@ -26,6 +26,9 @@
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.ShortEnumEqualityComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.Generic.LongEnumEqualityComparer<>))]
 [assembly:System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Collections.ListDictionaryInternal))]
+#if netcoreapp
```

Comment:
We need to remember to remove this (in master) when PN has the change, I guess we should have an issue

Reply:
and the skip on the test

Reply:
This doesn't exist in master. I added these lines here for 2.0.